### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by imhta**
A library app to add new books and toggle between read and unread status.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+const sidebarElement = document.getElementById('sidebar');
+const addNewBookBtn = document.getElementById('new-book-btn');
+const closeSidebarBtn = document.getElementById('close-btn');
+const contentElement = document.getElementById('content');
+const formElement = document.getElementById('new-book-form');
+
+const library = JSON.parse(localStorage.getItem('library')) || [];
+
+function Book([title, author, descript, status = 'not-read']) {
```
Comment (SunnyAsar) [inline js/main.js]: Hey guys, point 4 from requirements, a book should have `number of pages` property 

Screenshot 2019-11-20 at 4 11 47 PM

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  this.title = title;
+  this.author = author;
+  this.descript = descript;
+  this.status = status;
+}
+
+function addBooks(book) {
+  library.push(book);
+}
+
```
Comment (SunnyAsar) [inline js/main.js:20]: - lets add a means to remove the book as well

Screenshot 2019-11-20 at 3 59 10 PM

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hi @imhta  and @Mesi21 👋 
well done with the project so far. we have a couple of requirements to meet please see the comments for the details.
### UI fix
- There is a UI problem, when there are more than 5 books, the are not visible. the page or table doesn't scroll, let's fix this.

**please request another review when the updates are in place.**
Happy Coding.

File: style/style.css [hunk 3]
```diff
@@ -0,0 +1,147 @@
+body {
+  margin: 0px;
+  height: 100vh;
```
Comment (AdnanAfsari) [inline style/style.css:3]: Use `min-height` instead of height, i will solve a UI problem that i still there 😄

**Review by AdnanAfsari — APPROVED** [review]
**Status: Approved**

Hello @imhta @Mesi21 

Nice work 👏 . I am giving 👍 for thi project but there is very small fix- check my comment and made the correction 😄 .

Happy Coding
Adnan Afsari

**Pull request merged**